examples/custom-environment/lib.rs

```diff
@@ -0,0 +1,125 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Like I said in the other PR, examples should have pretty comprehensive docs

Reply:
:white_check_mark: